mlflow/transformers/__init__.py

```diff
@@ -2652,7 +2713,8 @@ def decode_audio(encoded):
     @staticmethod
     def _validate_str_input_uri_or_file(input_str):
```

Comment:
Could we keep this method as it is and move your check on the data into a new function? This function's name is just like checking a single input string.